WalletWasabi.Fluent/Controls/PrivacyContentControl.axaml

```diff
@@ -8,24 +8,25 @@
   <Style Selector="controls|PrivacyContentControl">
     <Setter Property="Template">
       <ControlTemplate TargetType="controls:PrivacyContentControl">
-        <Panel Background="Transparent">
+        <Panel>
```

Comment:
Why is this change?

Reply:
Because I need mouse hover to be detected for either the `PrivacyReplacement` or the actual `Content`, but not for the control itself, to fix the problem described in #9473 where hit testing is positive on empty space.

Reply:
makes sense, thanks!